src/Uno.UI/UI/Xaml/UIElement.cs

```diff
@@ -1096,7 +1096,7 @@ public void Arrange(Rect finalRect)
 			}
 
 			var layouter = ((ILayouterElement)fwe).Layouter;
-			layouter.Arrange(finalRect);
+			layouter.Arrange(finalRect.DeflateBy(fwe.Margin));
```

Comment:
Screenshot comparison will need to be revisited to make sure there are no regressions. But other than that, LGTM if everything is green!

Reply:
Agree

Reply:
I went through droid,ios,skia screenshots. They looks alright against last merged .dev-25 and my other pr #16564.